nova/nova/urls.py

```diff
@@ -1,5 +1,5 @@
 from django.conf.urls.static import static
-from django.urls import path
+from django.urls import path, re_path
```

Comment:
What's this for?

Reply:
When I was trying to add query parameters to the url, first I tried to do it with re_path but then I have forgotten to delete it. It is useless, I will delete it.